eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -126,26 +155,31 @@ void optimizesIfXmlIsAbsentInCache(@TempDir final Path tmp) {
     }
 
     @Test
-    void optimizesBecauseCacheIsExpired(@TempDir final Path tmp) throws IOException {
-        final XML outdated = OptCachedTest.program(ZonedDateTime.now().minusMinutes(1));
-        final XML updated = OptCachedTest.program(ZonedDateTime.now());
-        OptCachedTest.save(tmp, outdated);
-        MatcherAssert.assertThat(
-            "We expected that the program will be optimized because the cache is expired",
-            new OptCached(path -> updated, tmp).apply(outdated),
-            Matchers.equalTo(updated)
+    void optimizesBecauseCacheIsExpired(
+        @TempDir final Path cache,
+        @TempDir final Path dir) throws Exception {
+        final Path program = OptCachedTest.save(
+            dir,
+            OptCachedTest.program("new program")
+        );
+        OptCachedTest.setTime(dir, FileTime.fromMillis(System.currentTimeMillis()));
+        OptCachedTest.save(
+            cache,
+            OptCachedTest.program("old program")
+        );
+        OptCachedTest.setTime(
+            cache,
+            FileTime.fromMillis(System.currentTimeMillis() - 5000)
         );
-    }
-
-    @Test
-    void optimizesIfTimeIsNotSet(@TempDir final Path tmp) throws IOException {
-        final XML without = OptCachedTest.program();
-        final XML with = OptCachedTest.program(ZonedDateTime.now());
-        OptCachedTest.save(tmp, without);
         MatcherAssert.assertThat(
-            "We expected that the program will be optimized because the cache doesn't have time",
-            new OptCached(path -> with, tmp).apply(without),
-            Matchers.equalTo(with)
+            "We expected that the program will be optimized because the cache is expired",
+            new OptCached(
+                path -> OptCachedTest.program("new program"),
```

Comment:
@Yanich96 You rather often use "new program" in tests. Maybe it's better to create one more overloaded method `program()` without arguments which will call `program("new program")`?